addon/utils/validation.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember'
+const {isPresent} = Ember
+
+function isDomainValid (domain) {
+  if (!domain) {
+    return false
+  }
+
+  const [min, max] = domain
+  return isPresent(min) && !isNaN(min) && isPresent(max) && !isNaN(max)
```

Comment:
Can you add an Ember.Logger.debug here in the failure case with the value (just so we know why certain code blocks aren't executing)